src/components/Disputes/DisputesStateProvider.js

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react'
+import { useDisputesSubscription } from '../../hooks/subscription-hooks'
+
+const DisputesContext = React.createContext()
+
+function DisputesStateProvider({ children }) {
+  // const connectedAccount = useConnectedAccount()
+
+  const Provider = DisputesContext.Provider
```

Comment:
I think we can remove these two :slightly_smiling_face: (and use `<DisputesContext.Provider>` directly).

Also do we really need a provider here? Since we are only consuming `useDisputesSubscription()`.

Reply:
Initially thought that it would be useful but since we only subscribe to disputes it's not really needed in this case (Actually we weren't using it so i'm removing it)